generated/niscope/niscope/_grpc_stub_interpreter.py

```diff
@@ -0,0 +1,425 @@
+            self._client.GetEqualizationFilterCoefficients,
+            grpc_types.GetEqualizationFilterCoefficientsRequest(vi=self._vi, channel=channel, number_of_coefficients=number_of_coefficients),
+        )
+        return response.coefficients
+
+    def get_error(self):  # noqa: N802
+        response = self._invoke(
+            self._client.GetError,
+            grpc_types.GetErrorRequest(vi=self._vi),
+        )
+        return response.error_code, response.description
+
+    def import_attribute_configuration_buffer(self, configuration):  # noqa: N802
+        self._invoke(
+            self._client.ImportAttributeConfigurationBuffer,
+            grpc_types.ImportAttributeConfigurationBufferRequest(vi=self._vi, configuration=configuration),
+        )
+
+    def import_attribute_configuration_file(self, file_path):  # noqa: N802
+        self._invoke(
+            self._client.ImportAttributeConfigurationFile,
+            grpc_types.ImportAttributeConfigurationFileRequest(vi=self._vi, file_path=file_path),
+        )
+
+    def init_with_options(self, resource_name, id_query, reset_device, option_string):  # noqa: N802
+        response = self._invoke(
+            self._client.InitWithOptions,
+            grpc_types.InitWithOptionsRequest(resource_name=resource_name, id_query=id_query, reset_device=reset_device, option_string=option_string, session_name=self._grpc_options.session_name, initialization_behavior=self._grpc_options.initialization_behavior),
+        )
+        self._close_on_exit = response.new_session_initialized
+        return response.vi
+
+    def initiate_acquisition(self):  # noqa: N802
+        self._invoke(
+            self._client.InitiateAcquisition,
+            grpc_types.InitiateAcquisitionRequest(vi=self._vi),
+        )
+
+    def lock(self):  # noqa: N802
+        self._lock.acquire()
+
+    def probe_compensation_signal_start(self):  # noqa: N802
+        self._invoke(
+            self._client.ProbeCompensationSignalStart,
+            grpc_types.ProbeCompensationSignalStartRequest(vi=self._vi),
+        )
+
+    def probe_compensation_signal_stop(self):  # noqa: N802
+        self._invoke(
+            self._client.ProbeCompensationSignalStop,
+            grpc_types.ProbeCompensationSignalStopRequest(vi=self._vi),
+        )
+
+    def read(self, channel_list, timeout, num_samples):  # noqa: N802
+        response = self._invoke(
+            self._client.Read,
+            grpc_types.ReadRequest(vi=self._vi, channel_list=channel_list, timeout=timeout, num_samples=num_samples),
+        )
+        return response.waveform, [waveform_info.WaveformInfo(x) for x in response.wfm_info]
+
+    def reset_device(self):  # noqa: N802
+        self._invoke(
+            self._client.ResetDevice,
+            grpc_types.ResetDeviceRequest(vi=self._vi),
+        )
+
+    def reset_with_defaults(self):  # noqa: N802
+        raise NotImplementedError('reset_with_defaults is not supported over gRPC')
```

Comment:
Do you know why this method isn't supported?

Reply:
So for this one I'm guessing we skipped over it because it's marked obsolete in [hapigen](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=/src/niscope/niscopeBaseDistribution/niScopeAPIMetadata/source/niScopeAPIMetadata/base/functions/functions_obsolete_header.hapigen&version=GBmain&line=2133&lineEnd=2134&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) and the [header](https://p4.natinst.com/browser/perforce/build/exports/ni/nisc/niScopeHeaders/official/export/.latest/.latest/includes/niScopeHeaders/niScopeObsolete.h#line_361)

Reply:
That makes me wonder why we included it in the Python API. 🤷‍♂️